src/msg.c

```diff
@@ -1329,7 +1333,7 @@ static void unregister_callback(struct l_timeout *timeout, void *user_data)
 	device_forget_destroy(mydevice);
 }
 
-static void proxy_removed(const char *device_id, void *user_data)
+static void on_device_removed(const char *device_id, void *user_data)
```

Comment:
Do we have any case that needs a function return?
Such as adding the msg on the queue again.

Reply:
No. The thing should be removed even if it isn't connected.

Reply:
Ok